test/features/resource_availability.feature

```diff
@@ -197,24 +197,3 @@ Feature: Re-identification risk governance acknowledgement or Resource visibilit
         And I go to "/dataset/contains-de-identified-data-yes"
         Then I press the element with xpath "//a[@title='resource created by SysAdmin and is available']"
         Then I should not see "Re-identification risk governance acknowledgement/Resource visibility"
-
-
-    Scenario: Clean up
```

Comment:
should not have existed in first place
the contains-de-identified-data-yes should only be used once in one scenario. and technically should be dynamic created with a number on the end for that scenario to stop clashes like this.

Reply:
Well, the way this feature was built, it creates the dataset and then has unique resources for each scenario. Which should work, but we occasionally get errors suggesting that the dataset doesn't exist, which smells like test scenarios running out of order and purging the dataset too early.

Reply:
all tests should work independently due to having the ability to run in parallel or out of order.